system/views/formcontrols/iconpicker/index.cfm

```diff
@@ -0,0 +1,23 @@
+<cfscript>
+	inputName    = args.name         ?: "";
+	inputId      = args.id           ?: "";
+	inputClass   = args.class        ?: "";
+	placeholder  = args.placeholder  ?: "";
+	defaultValue = args.defaultValue ?: "";
+
+	value  = event.getValue( name=inputName, defaultValue=defaultValue );
+	if ( not IsSimpleValue( value ) ) {
+		value = "";
+	}
+
+	if ( IsDate( value ) ) {
```

Comment:
I think we shouldn't be looking for dates here? Perhaps this has been left after copy and paste?